OpenSim/Common/TRCFileAdapter.cpp

```diff
@@ -150,7 +174,13 @@ TRCFileAdapter::extendRead(const std::string& fileName) const {
     // the data container.
     std::size_t line_num{_dataStartsAtLine - 1};
     auto row = nextLine();
+
     while(!row.empty()) {
+        if (row.at(0).empty()) { //if no Frame# then ignore as empty elements
```

Comment:
What does "ignore as empty elements" mean?

What's the reason for this change? Isn't possible for a frame number to be missing but the rest of the row has valid data?

It would be good to document details like this in the class description for `TRCFileAdapter`, so a user can troubleshoot parsing issues.